google/cloud/spanner_dbapi/cursor.py

```diff
@@ -510,13 +510,17 @@ def __iter__(self):
             raise ProgrammingError("no results to return")
         return self._itr
 
-    def list_tables(self):
+    def list_tables(self, schema_name=""):
         """List the tables of the linked Database.
 
         :rtype: list
         :returns: The list of tables within the Database.
         """
-        return self.run_sql_in_snapshot(_helpers.SQL_LIST_TABLES)
+        return self.run_sql_in_snapshot(
+            sql=_helpers.SQL_LIST_TABLES,
+            params={"schema_name": schema_name},
```

Comment:
We are naming param key as `schema_name` here but `@table_schema` in _helpers.py. Isn't that incorrect?

Reply:
Oh, good catch. Yes, that is incorrect, and should also be `table_schema` here.

Reply:
Fixed